src/dodal/stubs/wrapped.py

```diff
@@ -0,0 +1,141 @@
+import itertools
+from typing import Annotated, Any, Mapping, Optional, TypeVar
+
+import bluesky.plan_stubs as bps
+from bluesky.protocols import Movable
+
+from dodal.common import MsgGenerator
+
+"""
+Wrappers for Bluesky built-in plan stubs with type hinting
+"""
+
+Group = Annotated[str, "String identifier used by 'wait' or stubs that await"]
+T = TypeVar("T")
+
+
+def set_absolute(
```

Comment:
Must: This is really horrible that we have to have this here. Is there a long term plan to add typing to `bluesky`?